bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "gpii-binder",
+  "version": "0.0.0",
+  "homepage": "https://github.com/GPII/gpii-binder",
+  "authors": [
+    "Tony <tony@raisingthefloor.org>"
+  ],
+  "description": "A Fluid component library that creates live bindings between model data and views.",
+  "moduleType": [
+    "amd"
```

Comment:
I don't believe you actually are an AMD module since there is not the proper boilerplate - just a standard node module